backend/aci/server/log_schema_filter.py

```
@@ -0,0 +1,196 @@
+    client_ip: str | None = None
+    user_agent: str | None = None
+    x_forwarded_proto: str | None = None
+    request_id: str | None = None
+    agent_id: str | None = None
+    org_id: str | None = None
+    api_key_id: str | None = None
+    project_id: str | None = None
+    user_id: str | None = None
+
+
+class TelemetryFieldsSchema(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)